src/components/BottomModal/BottomModal.js

```diff
@@ -25,26 +39,64 @@ class BottomModal extends React.Component {
       },
     } = e;
 
+    selectedItems.map(value => {
+      console.log(value.name === name ? '중복' : '와우');
+    });
+
     if (isRequired || isSelected) {
       this.setState({ isRequired: false, isSelected: false });
     }
 
     const selectedItemObj = {
       id: selectedItems.length,
       name,
-      price,
+      price: parseInt(price),
       quantity: 1,
     };
     const newSelectedItems = selectedItems.concat(selectedItemObj);
-    this.setState({ selectedItems: newSelectedItems });
+    this.setState({ selectedItems: newSelectedItems }, () => {
+      this.calculateMoney();
+      this.calculateQuantity();
+      this.validateItems();
+    });
   };
 
-  countQuantity = idx => {
+  calculateQuantity = () => {
     const { selectedItems } = this.state;
+    const totalCount = selectedItems.reduce((acc, cur) => {
+      return acc + cur.quantity;
+    }, 0);
+    this.setState({ totalCount });
+  };
+
+  calculateMoney = () => {
+    const { selectedItems } = this.state;
+    const totalMoney = selectedItems.reduce((acc, cur) => {
+      return acc + cur.price * cur.quantity;
+    }, 0);
+    this.setState({ totalMoney });
+  };
+
+  countQuantity = (idx, kind) => {
+    const { selectedItems } = this.state;
+    const sign = kind === 'plus' ? +1 : -1;
+    const newSelectedItems = selectedItems.map(value => {
+      return value.id === idx
+        ? {
+            ...value,
+            quantity: value.quantity + sign,
+          }
+        : { ...value };
+    });
+    this.setState({ selectedItems: newSelectedItems }, () => {
+      this.calculateMoney();
+      this.calculateQuantity();
+    });
```

Comment:
- 리뷰 사항 반영하면, 아래와 같이 수정 되어야 합니다!
```suggestion
    this.setState({ selectedItems: newSelectedItems });
```

Reply:
수정했습니다🤗